File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        //data structure to hold the integer array 

        //iterate through the values with a loop 

        //check for duplicates by

        //if arr[i] = arr[i-1]{return true}
        for(int i = 0; i < nums.length; i++){
            for(int j = 0; j < nums.length; j++){
                if(i != j && nums[i] == nums[j]){
                    return true;
                }
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        length = len(nums) 
        

        array = [[] for i in range(len(nums) * 2)] 

        for i in range(length):
            array[i] = nums[i]
        for i in range(length, len(array)):
            array[i] = nums[i-length]
        return array

        